extensions/-happen3-/additionalroots.js

```diff
@@ -0,0 +1,110 @@
+// Name: Additional Roots
+// ID: additionalroots
+// Description: A collection of new additional roots for your enjoyment.
+// By: -happen3-
+// Original: -happen3-
+
+// I wonder if this will ever pass the check?
+
+(function (Scratch) {
+  "use strict";
+
+  function AdditionalRoots(runtime) {
+    this.runtime = runtime;
+  }
+
+  AdditionalRoots.prototype.getInfo = function () {
+    return {
+      id: "additionalroots",
+      name: "Additional Roots!",
+      blocks: [
+        {
+          opcode: "squareRoot",
+          blockType: Scratch.BlockType.REPORTER,
+          text: "square root of [num]",
+          arguments: {
+            num: {
+              type: Scratch.ArgumentType.NUMBER,
+              defaultValue: 9,
+            },
+          },
+        },
+        {
+          opcode: "cubeRoot",
+          blockType: Scratch.BlockType.REPORTER,
+          text: "cube root of [num]",
+          arguments: {
+            num: {
+              type: Scratch.ArgumentType.NUMBER,
+              defaultValue: 8,
+            },
+          },
+        },
+        {
+          opcode: "fourthRoot",
+          blockType: Scratch.BlockType.REPORTER,
+          text: "fourth root of [num]",
+          arguments: {
+            num: {
+              type: Scratch.ArgumentType.NUMBER,
+              defaultValue: 16,
+            },
+          },
+        },
+        {
+          opcode: "fifthRoot",
+          blockType: Scratch.BlockType.REPORTER,
+          text: "fifth root of [num]",
+          arguments: {
+            num: {
+              type: Scratch.ArgumentType.NUMBER,
+              defaultValue: 32,
+            },
+          },
+        },
+        {
+          opcode: "nthRoot",
+          blockType: Scratch.BlockType.REPORTER,
+          text: "[n] to the power of 1/[nth]",
+          arguments: {
+            n: {
+              type: Scratch.ArgumentType.NUMBER,
+              defaultValue: 4,
+            },
+            nth: {
+              type: Scratch.ArgumentType.NUMBER,
+              defaultValue: 2,
+            },
+          },
+        },
+      ],
+      menus: {},
+      displayName: "Additional Roots!",
+    };
+  };
+
+  AdditionalRoots.prototype.squareRoot = function (args) {
```

Comment:
This goes for all functions- you're doing them wrong. It works, however it doesn't comply with the syntax. Please read the [extension making documentation](https://docs.turbowarp.org/development/extensions/introduction).